announcements/2020-Week-24.md

```diff
@@ -0,0 +1,29 @@
+# This Week in Releases (12.06.2020)
+
+1-**CoreOS is replaced with Flatcar Container Linux** in [AWS 9.0.6](https://github.com/giantswarm/releases/blob/master/aws/v9.0.6/release-notes.md)
```

Comment:
I would prefer a `1. ` (with a blank) over the `1-` without blank.

Reply:
Thanks! Done